crates/flowctl/src/collection/read/mod.rs

```diff
@@ -22,10 +22,6 @@ pub async fn get_collection_inferred_schema(
     ctx: &mut crate::CliContext,
     args: &SchemaInferenceArgs,
 ) -> anyhow::Result<()> {
-    if args.selector.exclude_partitions.len() > 0 || args.selector.include_partitions.len() > 0 {
```

Comment:
This was incorrect before this PR, so I just deleted it. The infer-schema subcommand is due for a bit of re-work anyway, since it should probably just fetch them from the control plane.